app/__mocks__/axios.js

```diff
@@ -0,0 +1,18 @@
+export default {
+  get: jest.fn().mockResolvedValue({
```

Comment:
This mock will return this data for all components that use Axios which isn't the best practice, can you add the mock only during the specific test?

Reply:
Sure, I think I wrote it in the test file as well:
[Test file- GuitarGallery.test.js, have a look at line: 8](https://github.com/LiorKGOW/My-Guitar-Store/blob/deb55b7f6dde33277bf3f217e8c4afb7e8fa72bb/test/javascript/GuitarGallery.test.js)

Do you think if I delete [this file (axios.js in the __mocks__ dir)](https://github.com/LiorKGOW/My-Guitar-Store/blob/deb55b7f6dde33277bf3f217e8c4afb7e8fa72bb/app/__mocks__/axios.js) it would be sufficient? 
I read that this is necessary in order to create a mock for it
Maybe if the file is empty that would take the implementation from the specific test?

Reply:
I think it can be simply: 
```js
export default {
  get: jest.fn(),
  post: jest.fn(),
  put: jest.fn(),
  delete: jest.fn(),
}